english/astral-project/astral-project.md

```diff
@@ -0,0 +1,11 @@
+First lie down, relax and focus your mind on one thing.
+
+Due to the relaxation and focus you'll start to notice the various sensations of your astral body
+splitting from your physical body. It might feel a little strange or scary but don't
+worry, this happens every night. Only then you are not aware of these sensations.
+
+Stay related, keep focused and enjoy this process of separation, if you stay with it you'll split.
+A voila, you have just arrived in the astral plane.
+If you did not make it or feel inspired ready more <a href="http://belsebuub.com/articles/experiences-in-dreams-and-in-the-astral-plane" target="_blank">here</a> to get inspired.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.